packages/flutter_test/lib/src/controller.dart

```diff
@@ -1243,6 +1385,8 @@ abstract class WidgetController {
   /// the widget's render object has been laid out at least once.
   Rect getRect(Finder finder) => Rect.fromPoints(getTopLeft(finder), getBottomRight(finder));
 
+  // TODO(pdblasi-google): Deprecate this and point references to semantics.get.
```

Comment:
I created #112670 to handle this. I wanted to keep this PR focused on the functionality, and I plan on fixing all instances of calls to `tester.getSemantics` when I actually deprecate this, which would muddy the PR.

Reply:
I actually prefer the other way around. If I am user I would rather interact with just one interface instead of more so that i don't need to read into the documentation of SemanticsController. I would prefer to make the entire SemanticsController private unless there is other use case.